lib/metrics.js

```diff
@@ -55,9 +59,3 @@ async function pushPrometheus() {
         promMetrics[key].reset();
     });
 }
-
-module.exports = {
-    reportCloudWatch,
-    pushPrometheus,
-    promMetrics,
-};
```

Comment:
This should not be removed, but rather be the default export to not change behaviour